consensus/hotstuff/verification/combined_verifier_v2.go

```diff
@@ -1,3 +1,4 @@
+//go:build relic
```

Comment:
Is this needed because of the use of [crypto.NewBLSKMAC](https://github.com/onflow/flow-go/pull/1601/files#diff-0cd49adfc8c2df8b741ff1252f2dc39210c56f5f9a506d21154e259b01b66cdeR40)?

If so, we need to add it for [staking signer as well](https://github.com/onflow/flow-go/pull/1601/files#diff-d8863e5cd779af826a1f1a6d5118d62905a89d3a7bdb5eb8833aa35e004ac51eR34)

Reply:
I think GoLand adds it automatically